deploy/kubernetes/tracee/tracee.yaml

```diff
@@ -57,6 +57,10 @@ spec:
         env:
           - name: LIBBPFGO_OSRELEASE_FILE
             value: /etc/os-release-host
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 3366
```

Comment:
Should this port be fixed?

Reply:
Yes, for the static installation (under deploy/kubernetes/tracee) it is always fixed, and matches the configmap we provide on the same file https://github.com/aquasecurity/tracee/blob/main/deploy/kubernetes/tracee/tracee.yaml#L21.

For helm, it is the default and is updated based on what input the user gives.